lib/pages/dashboard/dashoard_page.dart

```diff
@@ -22,17 +38,53 @@ class _DashboardPageState extends State<DashboardPage> {
       sliver: SliverList(
         delegate: SliverChildListDelegate.fixed([
           Paragraph(title: "Vos clubs"),
-          DashboardClubs(onClubChange: (club) => _selectCurrentClub(club)),
+          DashboardClubs(key: ValueKey("dashboard-clubs"), onClubChange: (club) => _selectCurrentClub(club)),
           Paragraph(title: "Vos équipes"),
-          if (_currentClub != null) DashboardClubTeams(club: _currentClub, onTeamFavoriteChange: (_) => _selectCurrentClub(_currentClub)),
+          if (_currentClub != null) _buildDashboardElement(_teamsElement),
```

Comment:
Even if the concept of using a `Function` instance as parameter is cool, I'm not sure that this code is easier to understand than inlining https://github.com/eric-taix/volley34-mobile/pull/79/files#diff-30d704deb5d63a4c0851bea4681f954cR51 here (especially if we agree that https://github.com/eric-taix/volley34-mobile/pull/79/files#diff-30d704deb5d63a4c0851bea4681f954cR70 is useless).

Reply:
I used that to encapsulate dashboard elements in a `BlocBuilder` widget. It permits to factorize the code but I can remove it and put two `BlocBuilder` widgets if you want.

Reply:
O, didn't realize the `BlocBuilder` 👍